tests/scryer/issues.rs

```diff
@@ -10,7 +10,7 @@ use serial_test::serial;
 fn call_0() {
     load_module_test(
         "tests-pl/issue831-call0.pl",
-        "   error(existence_error(procedure,call/0),call/0).\n",
+        "   error(existence_error(procedure,call/0),'.'(predicate-call/0,[])).\n",
```

Comment:
as soon as it works: `[predicate-call/0]`? (that's the *expectation*)

Reply:
The source of this error appears to be 

https://github.com/mthom/scryer-prolog/blob/e4d9692535c9dcffc09d58fe97fca4e0efe9de6b/src/machine/machine_state.rs#L575-L580

Reply:
In ab34905 I have now removed the predicate context for those errors.
I don't see any useful context being available.
Though it might be worth looking into adding context for goals evaluated as part of an `initialization/1` directive i.e. something like `[declaration-initialization/1, goal-Goal, file-File, line-Line]`